### ISSUE TITLE
False warning when combining parameters in numpy style doctring

### ISSUE THREAD

**Issue by matthiastjung** [PyCQA/pylint#3420]
### Steps to reproduce
With the following example code i encounter warnings that should not pop up.
It does not seem to be fixed in the preview release. I'm using the command line option load-plugins=pylint.extensions.docparams.

```python
"""Test module to show a suspected bug"""

def testpass(valx, valy, valz):
    """Test method that pylint is okay with

    Parameters
    ----------
    valx : int
        description of valx
    valy : int
        description of valy
    valz : int
        description of valz

    Return
    ------
    int
        sum of valx, valy, valz
    """
    return valx + valy + valz

def testpass2(valx, valy, valz):
    """Test method that pylint is okay with

    Parameters
    ----------
    valx, valy, valz : int
        description of valx, valy, valz

    Return
    ------
    int
        sum of valx, valy, valz
    """
    return valx + valy + valz

def testfail(valx, valy, valz, factor):
    """Test method that pylint is NOT okay with

    Parameters
    ----------
    valx, valy, valz : int
        description of valx, valy, valz
    factor: int
        description of factor

    Return
    ------
    int
        sum of valx, valy, valz multiplied by factor
    """
    return (valx + valy + valz) * factor

```
### Current behavior
The following warnings show up:
* "valx, valy, valz" missing in parameter documentation; pyling(missing-param-doc) [37, 1]
* "valx, valy, valz" missing in parameter type documentation; pyling(missing-param-doc) [37, 1]

### Expected behavior
No warnings should show up, as described on https://numpydoc.readthedocs.io/en/latest/format.html
"When two or more input parameters have exactly the same type, shape and description, they can be combined"

### pylint --version output
Instead of `pylint --version output` the command `python -m pylint --version` was used due to system setup. As I don't know if there are any hidden differences I decided to mention that.

pylint 2.4.4
astroid 2.3.3
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]

**Comment by mjstevens777** [comment]
On a related note, I get W9011 missing-return-type-doc for the following formats supported by numpy

```
Returns
-------
iterable : iterable object
```

```
Returns
-------
order : {'C', 'F', 'A'}
    Description of `order`.
```

**Comment by PCManticore** [comment]
Thanks, I can confirm it's a bug.
